tox.ini

```diff
@@ -29,6 +29,8 @@ passenv =
        AWS_SESSION_TOKEN
 setenv =
        COV_FAIL_UNDER = 74.00
+       WR_ENGINE = ray
+       WR_MEMORY_FORMAT = modin
```

Comment:
We don't necessarily need this, right? The default with `ray` and `modin` installed would still be distributed.

Reply:
Nop, not needed but it allows us to test that env variables are actually reas